Core/Font/PGF.cpp

```diff
@@ -704,25 +722,26 @@ void PGF::SetFontPixel(u32 base, int bpl, int bufWidth, int bufHeight, int x, in
 		}
 	case PSP_FONT_PIXELFORMAT_8:
 		{
-			Memory::Write_U8(pixelColor, framebufferAddr);
+			// FIXED: Clean 8-bit pixel writing without dithering artifacts
+			Memory::Write_U8(pixelColor & 0xFF, framebufferAddr);
```

Comment:
This is nonsense, pixelColor is already an u8, anding it with 0xFF isn't doing anything. Same below.